doc/api/child_process.markdown

```diff
@@ -755,7 +784,7 @@ grep.stdin.end();
 * `message` {Object}
 * `sendHandle` {Handle object}
 * `callback` {Function}
-* Return: Boolean
+* Return: Boolean | Promise
```

Comment:
A+, will change. This will likely influence how we approach this with streams as well, since the child process message functionality technically _should_ be a stream.